collector/meminfo_numa_linux.go

```diff
@@ -132,3 +149,29 @@ func parseMemInfoNuma(r io.Reader) (map[meminfoKey]float64, error) {
 
 	return memInfo, nil
 }
+
+func parseMemInfoNumaStat(r io.Reader) (map[string]float64, error) {
+  var (
+    numaStat = map[string]float64{}
+    scanner  = bufio.NewScanner(r)
+    re       = regexp.MustCompile("\\((.*)\\)")
+  )
+
+	for scanner.Scan() {
+		line := strings.TrimSpace(scanner.Text())
+		if line == "" {
+			continue
+		}
+		parts := strings.Fields(string(line))
+
+		fv, err := strconv.ParseFloat(parts[1], 64)
+		if err != nil {
+			return nil, fmt.Errorf("invalid value in meminfo: %s", err)
+		}
+
+		// Active(anon) -> Active_anon
+		metric := re.ReplaceAllString(parts[0], "_stat_${1}")
```

Comment:
The `numastat` file doesn't contain any round brackets i.e. this `re.ReplaceAllString()` is superfluous.